HW1/task1.py

```diff
@@ -0,0 +1,15 @@
+general_number = int(input("Введите трехзначное число "))
+number = general_number
+count = 0
+
+while number > 0:
+    count += 1
+    number = number // 10
```

Comment:
Я так понял этот цикл у вас направлен только на то, чтобы посчитать количество цифр. Но это все равно, что стрелять по мухе из пушки.
Можно сделать так:
```suggestion
count = len(str(number))
```